src/main/scala/AppenginePlugin.scala

```diff
@@ -2,64 +2,82 @@ package sbtappengine
-    lazy val devServer      = InputKey[revolver.AppProcess]("appengine-dev-server", "Run application through development server.")
-    lazy val stopDevServer  = TaskKey[Unit]("appengine-stop-dev-server", "Stop development server.")
-    lazy val enhance        = TaskKey[Unit]("appengine-enhance", "Execute ORM enhancement.")
-    lazy val enhanceCheck   = TaskKey[Unit]("appengine-enhance-check", "Just check the classes for enhancement status.")
 
-    lazy val onStartHooks   = SettingKey[Seq[() => Unit]]("appengine-on-start-hooks")
-    lazy val onStopHooks    = SettingKey[Seq[() => Unit]]("appengine-on-stop-hooks")
-    lazy val apiToolsJar    = SettingKey[String]("appengine-api-tools-jar", "Name of the development startup executable jar.")
-    lazy val apiToolsPath   = SettingKey[File]("appengine-api-tools-path", "Path of the development startup executable jar.")
-    lazy val sdkVersion     = SettingKey[String]("appengine-sdk-version")
-    lazy val sdkPath        = SettingKey[File]("appengine-sdk-path")
-    lazy val classpath      = SettingKey[Classpath]("appengine-classpath")
-    lazy val apiJarName     = SettingKey[String]("appengine-api-jar-name")
-    lazy val apiLabsJarName = SettingKey[String]("appengine-api-labs-jar-name")
-    lazy val jsr107CacheJarName = SettingKey[String]("appengine-jsr107-cache-jar-name")
-    lazy val binPath        = SettingKey[File]("appengine-bin-path")
-    lazy val libPath        = SettingKey[File]("appengine-lib-path")
-    lazy val libUserPath    = SettingKey[File]("appengine-lib-user-path")
-    lazy val libImplPath    = SettingKey[File]("appengine-lib-impl-path")
-    lazy val apiJarPath     = SettingKey[File]("appengine-api-jar-path")
-    lazy val appcfgName     = SettingKey[String]("appengine-appcfg-name")
-    lazy val appcfgPath     = SettingKey[File]("appengine-appcfg-path")
-    lazy val overridePath   = SettingKey[File]("appengine-override-path")
-    lazy val overridesJarPath = SettingKey[File]("appengine-overrides-jar-path")
-    lazy val agentJarPath   = SettingKey[File]("appengine-agent-jar-path")
-    lazy val emptyFile      = TaskKey[File]("appengine-empty-file")
-    lazy val temporaryWarPath = SettingKey[File]("appengine-temporary-war-path")
-    lazy val localDbPath    = SettingKey[File]("appengine-local-db-path")
-    lazy val debug          = SettingKey[Boolean]("appengine-debug")
-    lazy val debugPort      = SettingKey[Int]("appengine-debug-port")
-    lazy val includeLibUser = SettingKey[Boolean]("appengine-include-lib-user")
-    lazy val persistenceApi = SettingKey[String]("appengine-persistence-api", "Name of the API we are enhancing for: JDO, JPA.")
+  override def requires = sbt.plugins.JvmPlugin
+
+  object autoImport {
+    object AppengineKeys extends revolver.RevolverKeys {
+      lazy val requestLogs    = InputKey[Unit]("appengine-request-logs", "Write request logs in Apache common log format.")
```

Comment:
Key names should be renamed to `appengineRequestLogs` so they'll look the same on both build.sbt and the shell.